db_test.go

```diff
@@ -52,9 +78,12 @@ func TestDataAvailableOnlyAfterCommit(t *testing.T) {
 	querier = db.Querier(0, 1)
 	defer querier.Close()
 
-	seriesSet = querier.Select(labels.NewEqualMatcher("foo", "bar"))
-	if !seriesSet.Next() {
-		t.Fatalf("Error, was expecting a matching series.")
+	seriesSet = readSeriesSet(querier.Select(labels.NewEqualMatcher("foo", "bar")))
+	expected := map[string][]Sample{
+		`{foo="bar"}`: []Sample{{T: 0, V: 0}},
+	}
+	if !reflect.DeepEqual(seriesSet, expected) {
```

Comment:
So we are using the `require` package in a bunch of places in our projects by now.
Many people advocate against only using stdlib stuff, but practically, it does exactly the same and is less noisy. I think we should use it somewhat consistently.